src/main/java/ru/sbt/mipt/oop/Alarm/AlarmTest.java

```diff
@@ -0,0 +1,85 @@
+package ru.sbt.mipt.oop.Alarm;
+
+import org.junit.jupiter.api.Test;
+import ru.sbt.mipt.oop.Room;
+import ru.sbt.mipt.oop.RoomObjects.Door;
+import ru.sbt.mipt.oop.RoomObjects.Light;
+import ru.sbt.mipt.oop.SmartHome;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class AlarmTest {
+    Light light1 = new Light("1", "LIGHT_OFF");
+    Light light2 = new Light("2", "LIGHT_ON");
+    Door door1 = new Door("3", "DOOR_OPEN");
+    List<Door> doors = Arrays.asList(door1);
+    List<Light> lights = Arrays.asList(light1, light2);
+    Room room = new Room(lights, doors, "hall");
+    List<Room> roomList = Arrays.asList(room);
+    SmartHome smartHome = new SmartHome(roomList);
+    Alarm alarm = new Alarm(smartHome);
+
+
+    @Test
+    void AlarmStandardCreationTypeOfAlarm() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.